api/v1alpha1/olsconfig_types.go

```diff
@@ -56,6 +56,9 @@ type OLSSpec struct {
 	// +kubebuilder:validation:Enum=DEBUG;INFO;WARNING;ERROR;CRITICAL
 	// +kubebuilder:default=INFO
 	LogLevel string `json:"logLevel,omitempty"`
+	// Disable TLS for OLS server. Default: "false"
+	// +kubebuilder:default=false
+	DisableTLS bool `json:"disableTLS,omitempty"`
```

Comment:
@vbelouso I dont think we need this control for the user. @bparees you want this for any reason ?

Reply:
nope, you can delete it.  OLS, when managed by the operator, should always secure its endpoints with TLS